tests/run-pass/aux_test.rs

```diff
@@ -1,4 +1,5 @@
 // aux-build:dep.rs
+// This ignores the test against rustc, but runs it against miri:
```

Comment:
If I comment out the `// ignore-cross-compile` line and run `cargo test` locally (on Linux), the test passes.